aiodocker/docker.py

```diff
@@ -46,6 +47,14 @@ def __init__(self,
             elif url.startswith('/'):
                 connector = aiohttp.connector.UnixConnector(url)
                 self.url = "http://docker" #aiohttp treats this as a proxy
+            elif url.startswith('tcp://') and os.environ.get('DOCKER_TLS_VERIFY', None) == '1':
+                connector = aiohttp.TCPConnector(ssl_context=self._docker_machine_ssl_context())
+                self.url = url.replace('tcp://', 'https://')
+
+            elif url.startswith('tcp://') and os.environ.get('DOCKER_TLS_VERIFY', None) == '0':
+                connector = aiohttp.TCPConnector()
+                self.url = url.replace('tcp://', 'http://')
+
```

Comment:
The code seems to be redundant because the only difference between two elif clauses are `ssl_context` argument which depends on `DOCKER_TLS_VERIFY` environment variable. It would better to refactor so that `ssl_context` argument is assigned None or the value depending on the result of checking `os.environ`.